src/components/application_manager/test/application_manager_impl_test.cc

```diff
@@ -269,6 +269,18 @@ class ApplicationManagerImplTest
     ASSERT_TRUE(mock_app_ptr_.get());
   }
 
+  void SetUpRequestForInterfacesAvailabilityExpectation(
+      const hmi_apis::FunctionID::eType function_id) {
+    smart_objects::SmartObject sm_object(smart_objects::SmartType_Map);
+    sm_object[strings::params][strings::function_id] =
+        static_cast<int>(function_id);
+    smart_objects::SmartObjectSPtr sptr =
```

Comment:
```suggestion
    const auto sptr =
```

Reply:
@mvorobio thank you , [done ](https://github.com/LuxoftSDL/sdl_core/pull/38/commits/5c63e9b59932b7f628c6aee0592f38ba0481d911)